### PR TITLE
rosegarden: init at 14.12

### PR DESCRIPTION
**Pull request by lebastr**

### CODE REVIEW

**Comment by lebastr** [comment]
Travis said: The build failed due to running out of memory. What to do next?

File: pkgs/applications/audio/rosegarden/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, qt4, pkgconfig, ladspaPlugins, ladspaH,
+  dssi, liblo, liblrdf, fftwSinglePrec, libsndfile,
+  libsamplerate, perl, makedepend, libjack2,
+  withLirc ? false, lirc ? null } :
+
+let rosegarden = rec {
```
Comment (lethalman) [inline pkgs/applications/audio/rosegarden/default.nix]: Can you please put everything into the derivation instead of using this let? It's pointless.

Comment (lebastr) [inline pkgs/applications/audio/rosegarden/default.nix]: I've done it.

**Comment by lethalman** [comment]
Built locally thanks.

**Pull request merged**